--- /usr/bin/adt-buildvm-ubuntu-cloud	2016-04-19 08:08:42.000000000 +0200
+++ adt-buildvm-ubuntu-cloud	2016-12-23 12:12:40.666569441 +0100
@@ -148,7 +148,10 @@
 
 
 def download_image(cloud_img_url, release, arch):
-    diskname = '%s-server-cloudimg-%s-disk1.img' % (release, arch)
+    if release in ('zesty', 'yakkety'):
+        diskname = '%s-server-cloudimg-%s.img' % (release, arch)
+    else:
+        diskname = '%s-server-cloudimg-%s-disk1.img' % (release, arch)
     image_url = '%s/%s/current/%s' % (cloud_img_url, release, diskname)
     local_image = os.path.join(workdir, diskname)
 
